src/cljs/rems/navbar.cljs

```diff
@@ -62,16 +62,16 @@
   ;;TODO: get navigation options from subscription
   (let [roles (:roles identity)]
     [e (into [:div.navbar-nav.mr-auto
-              (when (roles/is-logged-in? roles)
-                [nav-link "/catalogue" (text :t.navigation/catalogue)])
+              (when-not (:user identity)
+               [:<> [nav-link "/" (text :t.navigation/home) :exact]
+                [nav-link "/catalogue" (text :t.navigation/catalogue)]])
```

Comment:
This is still missing the check for the config. It should only be shown if it's public.